juntagrico/forms.py

```diff
@@ -38,6 +38,39 @@ def clean_password_repeat(self):
         return self.data['passwordRepeat']
 
 
+class MemberCancellationForm(Form):
```

Comment:
Thinking about it, the name and address of the member should also be in this form, to be confirmed/updated, as it is needed for the template and must match the bank information.

Reply:
added